src/meta/src/manager/id.rs

```diff
@@ -211,6 +213,7 @@ where
             IdCategory::Database => &self.database,
             IdCategory::Schema => &self.schema,
             IdCategory::Table => &self.table,
+            IdCategory::Sink => &self.sink,
```

Comment:
why CI relate to meta

Reply:
hmm this is still leftover code from ns-sink-meta before it got merged with main. I will correct it 👍